src/frontend/lib/components.ts

```diff
@@ -0,0 +1,16 @@
+import {
+  ActionButton,
+  type ActionButtonProps
+} from './components/buttons/ActionButton';
+
+import { Boundary } from './components/Boundary';
+import { ButtonMenu } from './components/buttons/ButtonMenu';
+import { CopyButton } from './components/buttons/CopyButton';
```

Comment:
Just as a shortcut, we could also use 
```suggestion
export { CopyButton } from './components/buttons/CopyButton';
```

Reply:
Ok, didn't know that - thanks!